lib/src/components/NxList/NxListSubtext.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import classnames from 'classnames';
+import {NxListSubtextProps, nxListSubtextPropTypes} from './types';
+
+const NxListSubtext = forwardRef<HTMLSpanElement, NxListSubtextProps>((props: NxListSubtextProps, ref) => {
```

Comment:
`withClass`

Reply:
> `withClass`

Converted to use `withClass`.
88365303aa6c6cd441566999a777569fa0022ec3